unicodetools/src/main/java/org/unicode/props/UcdProperty.java

```diff
@@ -281,6 +281,8 @@ public enum UcdProperty {
     IDS_Binary_Operator(PropertyType.Binary, Binary.class, null, "IDSB"),
     IDS_Trinary_Operator(PropertyType.Binary, Binary.class, null, "IDST"),
     IDS_Unary_Operator(PropertyType.Binary, Binary.class, null, "IDSU"),
+    ID_Compat_Math_Continue(PropertyType.Binary, Binary.class, null, "ID_Compat_Math_Continue"),
```

Comment:
@eggrobin did you get this from running GenerateEnums?

Reply:
All changes are manual, but I think this one is checked by tests.